src/conways.rs

```diff
@@ -0,0 +1,126 @@
+const COLUMNS: usize = 32;
+const ROWS: usize = 32;
+
+#[derive(Clone, Copy, PartialEq, Debug)]
+pub enum CellState {
+    Alive,
+    Dead,
+}
+
+#[derive(Clone, Copy)]
+pub struct Position(pub usize, pub usize);
+
+pub struct Conways {
+    pub grid: Vec<Vec<CellState>>,
```

Comment:
I don't think a Struct when you have just 1 member makes too much sense when you could've written just functions that take a `type Grid = Vec<Vec<CellState>>` for example, maybe if you would've had game state, etc. it would've made more sense. But it's OK you don't have to change it.